flask_celeryext/app.py

```diff
@@ -3,39 +3,28 @@
 # This file is part of Flask-CeleryExt
 # Copyright (C) 2015-2019 CERN.
 # Copyright (C) 2018-2019 infarm - Indoor Urban Farming GmbH.
+# Copyright (C) 2022 Graz University of Technology.
 #
 # Flask-CeleryExt is free software; you can redistribute it and/or modify it
 # under the terms of the Revised BSD License; see LICENSE file for more
 # details.
 
 """Flask extension for Celery integration."""
 
-from __future__ import absolute_import, print_function
-
-import warnings
-from distutils.version import LooseVersion
-
 import flask
 from celery import Task
-from celery import __version__ as celery_version
 from celery import current_app as current_celery_app
 from celery import signals
 
-from ._mapping import V3TOV4MAPPING
+from ._mapping import INVENIO_TO_CELERY_MAPPING
```

Comment:
Comment: I would name it ``FLASK_TO_CELERY_MAPPING`` instead, since the module is usable outside of Invenio (similar other places where Invenio is used).